content/CORE/CORETutorials/Storage/Disks/DiskReplace.md

```diff
@@ -79,3 +79,65 @@ For pools with large amounts of data, this can take a long time.
 When the resilver is complete, the pool status screen updates to show the new disk and the pool status returns to **Online**.
 
 ![Storage Pools Status Replace Complete](/images/CORE/12.0/StoragePoolsStatusReplaceComplete.png "Replacement Complete")
+
+## CLI Method
```

Comment:
@Mrt134 did you mean to remove this here and leave on release notes only?

Reply:
Whoops, forgot to hit save on that file 🤦‍♂️